ifrs17/Import/3ImportScope-Actuals.ipynb

```diff
@@ -173,7 +184,7 @@
                 "\n    [IdentityProperty][NotVisible][Dimension(typeof(EstimateType))]",
                 "\n    string EstimateType => EstimateTypes.A;",
                 "\n    ",
-                "\n    private int?[] accidentYears => GetStorage().GetAccidentYears(Identity.DataNode).ToArray();",
+                "\n    private int?[] accidentYears => GetStorage().GetAccidentYears(Identity.DataNode, Identity.ProjectionPeriod).ToArray();",
                 "\n    ",
                 "\n    [NotVisible]",
                 "\n    ActualBase[] Actuals => GetScope<ValidAmountType>(Identity.DataNode).ActualAmountTypes",
```

Comment:
You need to concat the be AmountTypes here

Reply:
not necessary for CSM

Reply:
Yes, as discussed, we can do it later.